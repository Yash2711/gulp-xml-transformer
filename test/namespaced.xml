<?xml version="1.0" encoding="UTF-8"?>
<root xmlns="https://github.com/dotcore64/gulp-xml-transformer">
  <name>this is name</name>
  <version major="1" minor="0"></version>
</root>
