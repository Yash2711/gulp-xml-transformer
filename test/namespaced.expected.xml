<?xml version="1.0" encoding="UTF-8"?>
<root xmlns="https://github.com/dotcore64/gulp-xml-transformer">
  <name>new name</name>
  <version major="1" minor="0"/>
</root>
